lib/jsonapi/utils/param_parser.ex

```diff
@@ -0,0 +1,31 @@
+defprotocol JSONAPI.Utils.ParamParser do
+  @fallback_to_any true
+  def parse(params)
+end
+
+defimpl JSONAPI.Utils.ParamParser, for: Any do
+  def parse(data), do: data
+end
+
+defimpl JSONAPI.Utils.ParamParser, for: List do
+  def parse(list), do: Enum.map(list, &JSONAPI.Utils.ParamParser.parse/1)
+end
+
+# Pass built in data types through
+defimpl JSONAPI.Utils.ParamParser,
```

Comment:
Do we need this implementation or the one below? Since we're using `@fallback_to_any` and all of these just pass though, is it necessary?